partners/arrow-markets.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "Arrow Markets",
+    "description": "Arrow Markets democratizes options trading through a hybrid RFQ system with on-chain exchange and settlement.",
+    "imgUrl": "https://x.com/ArrowMarkets/photo",
+    "missions": [
+      {
+        "name": "Arrow Markets_",
+        "description": "Complete liquidity provision and purchasing option on Arrow Markets",
+        "imgUrl": "https://img5.pic.in.th/file/secure-sv1/mission1.jpg",
+        "tags": ["DeFi"],
+        "quests": [
+          {
+            "name": "Liquidity Provision",
+            "description": "Provide Liquidity on Trader Joe for the arrow market pool",
+            "category": "DeFi",
+            "contract_address": "0x7F91272ff1A0114743D2df95F5905F9613Fd92b3",
+            "rewards": [
+              {
+                "type": "points",
+                "value": 60
+              }
+            ],
+            "tasks": [
+              {
+                "name": "Provide Liquidity",
+                "type": "POOL",
+                "description": "Provide Liquidity for Arrow Token on Trader Joe",
+                "contract_address": "0xc3311F358AFaD52e13425d41053F6fdCaf0DE6dB"
+              }
+            ]
+          },
+          {
+            "name": "Option Purchase",
+            "description": "Purchase any option on the arrow market platform",
+            "category": "DeFi",
+            "rewards": [
+              {
+                "type": "points",
+                "value": 60
+              }
+            ],
+            "tasks": [
+              {
+                "name": "Perform Trade",
+                "type": "TRADE",
+                "description": "Purchase an option on the arrow market platform"
```

Comment:
Since this is an on-chain activity, a contract_address, and method_id should be provided

Reply:
the way our contract works is that we have a router that acts on the behalf of the user so it won't be a direct contract interaction.

Here is a sample transaction on testnet:
https://testnet.snowtrace.io/tx/0xfb004853f9f9cf96e0a28d25367838aa5514e044df735e9074c18ff8fe7a4cc4

where here the user is address `0x8fcD6555eF027772e4AAf0B8aEBf36e47aff9E16`

Let me know how you want to work with this

Reply:
Is there a mainnet sample you can send?

Reply:
https://snowtrace.io/tx/0x080e3285542c69b77e75f5d85409b239f025392b6ac57fd78a8c76cf65d51471

Reply:
In the case of this transaction, the contract_address is 0xC33867F01E320bB077415cD3592E6B3879AaC45d and the method_id is 0xc593dd27, you can get that from snowtrace as well

Reply:
to confirm, the user `0x8fcD6555eF027772e4AAf0B8aEBf36e47aff9E16` doesn't directly interact with the contract as you can see in the transaction, is this fine?

Reply:
Yes it is, the user details have, particularly the wallet address have to be forwarded to the router to handle the transaction